static/js/components/dashboard/CourseRow.js

```diff
@@ -206,11 +206,8 @@ export default class CourseRow extends React.Component {
   }
   getCourseTag = (): React$Element<*> => {
     const { course } = this.props
-    return (
-      <div className="elective-tag">
-        {course.is_elective ? "Elective" : "Core"}
-      </div>
-    )
+    const tag = course.is_elective ? "Elective" : "Core"
+    return <div className={`elective-tag ${tag}`}>{tag}</div>
```

Comment:
Do we need the `tag` in the `className` here? We should remove it if not. It doesn't look like there is a CSS class for these names, and if so it should not contain capital letters.

Reply:
yes, there is a class that corresponds to the tag name.